Code/Engine/Foundation/Memory/Policies/AllocPolicyStack.h

```diff
@@ -3,11 +3,12 @@
 #include <Foundation/Containers/HybridArray.h>
 
 /// \brief This allocation policy works like a stack. You can only "push" and "pop" allocations
-///   in the correct order.
+///   in the correct order. The allocation policy can poison all allocations to show usages after destruction.
```

Comment:
"poison" sounds dangerous although it is supposed to help the user. Maybe something like "OverwriteMemoryOnReset" would be more descriptive?

Reply:
Fixed